joomla/joomla-cms
# [4.0] Use preload instead of prefetch alternative

Pull Request for Issue # .

### Summary of Changes
Prefetch = load an asset for the next navigation (this is very low priority)
Preload = load an asset with higher priority for the current page

In short, proload is what is needed for the css (color, fonts)

This PR is essentially is implementing basically the https://css-tricks.com/the-fastest-google-fonts/ with some extras:
- the webfont will also be available even if Javascript is disabled (the same thing I did for font awesome)
- the preload is done in the actual headers of the response
- Inlined the CSS Properties for all the fonts
- The code is written in a way that a custom google font URL can be used (not exposed to UI, but a simple input and a toggle button will do the trick)

- There are changes in the SQL that are not automatically applied
- There are SCSS files that were deleted

### Testing Instructions

Apply the PR, and make sure to save the Cassiopeia default style!!!
Check all the fonts and the loading (preferably by emulating a very slow connection)

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
